benchmark/config.js

```diff
@@ -0,0 +1,15 @@
+const { resolve } = require('path')
+const { existsSync } = require('fs')
+
+const config = resolve(`${__dirname}/../benchmark-config.js`)
+
+if (existsSync(config)) {
+  module.exports = require(config)
+} else {
+  module.exports.include = {
+    slate: /^/,
+    'slate-html-serializer': /^/,
+    'slate-plain-serializer': /^/,
+    'slate-react': /^/,
+  }
+}
```

Comment:
I am wondering whether the grep is too long for writing in command line?

Reply:
I don't want to have to edit files for that kind of thing. I'd like either `--match <string>` for both commands. Or, if we wanted to get smarter, it could know that running "compare" can only compare against the benchmarks that have already been "saved" so you wouldn't need to add it twice.